packages/notifi-react-card/lib/components/subscription/subscription-card-views/AlertListPreview.tsx

```diff
@@ -47,7 +47,7 @@ export const AlertListPreview: React.FC<AlertListPreviewProps> = ({
           )}
           key={eventType.name}
         >
-          {eventType.name}
+          {eventType.type === 'fusion' ? (eventType.displayNameOverride ?? eventType.name) : eventType.name}
```

Comment:
don't we also have fusionHealthCheck

Reply:
that is just a sub type of `fusion` i think